solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/SwapCoresPayload.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.solrj.request.beans;
+
+import org.apache.solr.common.annotation.JsonProperty;
+import org.apache.solr.common.util.ReflectMapWriter;
+
+public class SwapCoresPayload implements ReflectMapWriter {
+    @JsonProperty(required = true)
```

Comment:
Should we be doing `required = true` is other places as part of this conversion?

Reply:
Seems like having some validation on these would be a big win if it's built in to this style of API?

Reply:
Yep, this enforcement is built in to the annotation-style APIs.

> Should we be doing required = true is other places as part of this conversion?
For sure - if there's a place where I skipped `required = true` that's actually required, absolutely lmk.  I thought I set it everywhere I should have, but it's easy to miss.